docs/api-reference/arcgis/arcgis-deck-layer.md

```diff
@@ -31,8 +32,26 @@ loadArcGISModules().then(({ArcGISDeckLayer}) => {
 const layer = new ArcGISDeckLayer(props)
 ```
 
-`props` are forwarded to a `Deck` instance. The following [Deck](/docs/api-reference/deck.md) props are supported:
-
-- `deckLayers` is forwarded to `layers`
-- `effects`
-- `parameters`
+Property names that start with `deck.` are forwarded to a `Deck` instance. The following [Deck](/docs/api-reference/deck.md) props are supported:
```

Comment:
In this case, whatever the user passes to the `DeckLayer` constructor can be modified as such:

```js
const arcGISLayer = new DeckLayer({
  'deck.layers': [...]
});

// later in the app
arcGISLayer.deck.layers = [...];
```

This is aligned with how the other layers in ArcGIS library work. Since this module is an implementation of the ArcGIS API, I think it's reasonable to stick to ArcGIS's conventions and expectations.